CMakeLists.txt

```diff
@@ -295,7 +269,9 @@ list(APPEND fms_header_files
 # Standard FMS compiler definitions
 list(APPEND fms_defs
   use_libMPI
-  use_netCDF)
+  use_netCDF
+  SPMD
+  __IFC)
```

Comment:
Just wandering what these definitions are. No worries if you don't know.

Reply:
I don't know. I copied them from https://github.com/ACCESS-NRI/FMS/blob/98a5532b2e261e408ed0f32e0738fc601bd96594/CMakeLists.txt#L70 .